dfaligner/cli.py

```diff
@@ -187,7 +187,7 @@ def extract_alignments(
                 save_dir
                 / "text_grid"
                 / SEP.join([basename, speaker, language, "duration.TextGrid"]),
-                text_processor.token_sequence_to_text_sequence(tokens.tolist()),
+                text_processor._token_sequence_to_text_sequence(tokens.tolist()),
```

Comment:
Change the name `_token_sequence_to_text_sequence` as the leading underscore implies a private method as stated in https://pep8.org/#descriptive-naming-styles